src/components/page/elements/ScriptElement.js

```diff
@@ -1,41 +1,45 @@
-import shajs from 'sha.js';
+import { getSHA256 } from '../../../lib/helpers'
 
 export default function ScriptElement(props) {
-    handleScriptContent(props.content);
-    handleScriptSrc(props.element.src);
-    return (null);
+  handleScriptContent(props.content)
+  handleScriptSrc(props.element.src)
+  return null
 }
 
 function getElementByHash(value) {
-    const scriptContentHash = shajs('sha256').update(value).digest('hex');
-    const doesAlreadyExist = document.getElementById(scriptContentHash);
-    return {hash: scriptContentHash, exists: doesAlreadyExist}
+  const scriptContentHash = getSHA256(value)
+  const doesAlreadyExist = document.getElementById(scriptContentHash)
+  return { hash: scriptContentHash, exists: doesAlreadyExist }
 }
 
 function handleScriptContent(content) {
-    if (content) {
-        content.forEach((itm) => {
-            if (itm !== "") {
-                const element = getElementByHash(itm);
-                if (!element.exists) {
-                    const script = document.createElement("script");
-                    script.id = element.hash;
-                    script.innerHTML = itm;
-                    document.body.appendChild(script);
-                }
-            }
-        });
-    }
+  if (content) {
+    content.forEach(itm => {
+      if (itm !== '') {
+        const element = getElementByHash(itm)
+        if (!element.exists) {
+          const script = document.createElement('script')
+          script.id = element.hash
+          script.innerHTML = itm
+          document.body.appendChild(script)
+        }
+      }
+    })
+  }
 }
 
 function handleScriptSrc(src) {
```

Comment:
Same here, negating assertions would flat down the structure of the function

Reply:
👍